ccm_web/server/src/user/user.model.ts

```diff
@@ -20,7 +20,8 @@ interface UserCreationAttributes extends Optional<UserAttributes, 'id'> {
 // Sequelize Datatypes mapping with Mysql https://sequelize.org/master/manual/model-basics.html#data-types
 @Table({
   tableName: 'user',
-  freezeTableName: true
+  freezeTableName: true,
+  underscored: true
```

Comment:
Why is `underscored` necessary here?  The table name doesn't include underscores.  Must this be specified if the columns within the table have names with underscores?

Reply:
You know, maybe it's not. I will try removing it and see what happens. The documentation for Sequelize says that it will convert camelCase attribute names to snake_case ones, but I'm already doing that manually via the `field` in the migrations. I guess I was thinking that this would give the model more information, allow for the correct traversal between snake_case tables/fields and [cC]amelCase models and attributes, but I'm questioning that now.
https://sequelize.org/master/manual/naming-strategies.html

Reply:
I was going to check out your branch to try it, but based on [your comment](https://github.com/tl-its-umich-edu/canvas-course-manager-next/pull/127#discussion_r658826076), I'll hold off until you've experimented a bit more with combinations of the `underscored` and name properties.

Speaking of branches, I thought **_I_** had a lot of old branches in my fork.  You win. 🏆

Reply:
Okay, @lsloan, I did some research and experimented. `underscored` seems to have a long history of being buggy in `sequelize` (see [this issue](https://github.com/sequelize/sequelize/issues/11225)). Based on my current understanding, I think we can go without specifying it and prefer the hardcoded approach using `field`. See https://github.com/tl-its-umich-edu/canvas-course-manager-next/pull/127/commits/00327465fb1ec84dd9dbad1561d0e7bba7448831

I do have a question about how to handle all this with `session`, which I'll comment on in the thread off of `app.module.ts`.

Reply:
Some additional explanation: we could have tried to lean into `underscored` and not use the `field` approach. However, when I did this, I didn't see that the custom-defined fields like `User.lastName` followed the snake_case naming convention. So I fell back to using `field`, which seems more predictable.